### DISCUSSION TITLE
Installation guide for Roborock S7

### DISCUSSION THREAD

**Discussion by mu88** [Hypfer/Valetudo#1782] [Q&A / Support] [answered]
Hi 👋🏻

I want to install Valetudo on a Roborock as S7 and have a question about the [installation guideline](https://valetudo.cloud/pages/installation/roborock.html):
> Connect the marked TPA17 to GND using your jumper cable or paperclip. You can use anything that is GND. The marked SH1 for GND should be close enough to enable you to do the procedure without the help of another person. There might be some conformal coating on TPA17. You can use your fingernails to carefully scrape that off.  
Press the power button for 3 seconds and keep the TPA17 connected to GND for 5 more seconds after that.

Both the textual instruction and the image are introducing _SH1_- however, I don't find any instruction where SH1 is used/connected, only TPA17.

So my question: is there something missing in the manual? For what do I need SH1 and when?

Thank you!

**Comment by Hypfer** [comment, marked as answer]
SH1 is just one of a few things that are connected to GND. You can use any GND but that's one that is easily accessible.

**Reply by mu88** [reply]
Ah I see, SH1 acts as GND 💡 so instead of searching a spot in my flat that I can use as GND for TPA17, I could use SH1, correct?

**Reply by Hypfer** [reply]
Do keep in mind that rooting in general but also specifically for the S7 is an advanced topic that requires some amount of prior knowledge in e.g. tinkering with electronics.

If you're new to this, I'd recommend not starting to learn it with this project.
Maybe you have a friend with more experience that might be able to help out? There are also many hackerspaces all over the world that are usually pretty open and willing to help newcomers. Maybe you have something nearby?

**Reply by mu88** [reply]
I'll see whom I can find, thank you!
